packages/renderer/src/lib/image/ImagesList.svelte

```diff
@@ -280,6 +284,13 @@ const row = new Row<ImageInfoUI>({
     {#if $imagesInfos.length > 0}
       <Prune type="images" engines="{enginesList}" />
     {/if}
+    <Button
+      on:click="{() => loadImages()}"
+      title="Load image(s) from tar archive(s)"
+      icon="{faArrowCircleDown}"
```

Comment:
Save used faDownload, seems like this should use upload (or find another pair so that the actions are mirrored).